src/Controls/src/Core/Compatibility/Handlers/ListView/Android/ViewCellRenderer.cs

```diff
@@ -307,7 +307,14 @@ protected override void OnMeasure(int widthMeasureSpec, int heightMeasureSpec)
 					}
 
 					var size = _viewHandler.MeasureVirtualView(widthMeasureSpec, heightMeasureSpec);
-					height = (int)size.Height;
+					if (_viewCell.Height > 0)
```

Comment:
If for example, tapping a button updates the source and causes an InvalidateMeasure of the cell, does it update the size or keep the previous size?

Reply:
@jsuarezruiz , if height for the view cell is explicitly defined, the cell will retain its specified height. If the height is not defined, it will update its size.